testnet/stacks-node/src/config.rs

```diff
@@ -25,6 +25,7 @@ pub struct ConfigFile {
     pub burnchain: Option<BurnchainConfigFile>,
     pub node: Option<NodeConfigFile>,
     pub mstx_balance: Option<Vec<InitialBalanceFile>>,
+    pub mstx_vesting_schedule: Option<Vec<InitialVestingScheduleFile>>,
```

Comment:
Might want to make these `ustx_balance` and `ustx_vesting_schedule` -- these are microSTX, not milliSTX